src/snowflake/snowpark/modin/plugin/docstrings/series_utils.py

```diff
@@ -1723,7 +1723,74 @@ def strftime():
         pass
 
     def round():
-        pass
+        """
+        Perform round operation on the data to the specified freq.
+
+        Parameters
+        ----------
+        freq : str or Offset
+            The frequency level to round the index to. Must be a fixed frequency like ‘S’ (second) not ‘ME’ (month end). See frequency aliases for a list of possible freq values.
+        ambiguous : ‘infer’, bool-ndarray, ‘NaT’, default ‘raise’
+            Only relevant for DatetimeIndex:
+            - ‘infer’ will attempt to infer fall dst-transition hours based on order
+            - bool-ndarray where True signifies a DST time, False designates a non-DST time (note that this flag is only applicable for ambiguous times)
+            - ‘NaT’ will return NaT where there are ambiguous times
+            - ‘raise’ will raise an AmbiguousTimeError if there are ambiguous times.
+        nonexistent : ‘shift_forward’, ‘shift_backward’, ‘NaT’, timedelta, default ‘raise’
+            A nonexistent time does not exist in a particular timezone where clocks moved forward due to DST.
+            - ‘shift_forward’ will shift the nonexistent time forward to the closest existing time
+            - ‘shift_backward’ will shift the nonexistent time backward to the closest existing time
+            - ‘NaT’ will return NaT where there are nonexistent times
+            - timedelta objects will shift nonexistent times by the timedelta
+            - ‘raise’ will raise an NonExistentTimeError if there are nonexistent times.
+
+        Returns
+        -------
+        DatetimeIndex, TimedeltaIndex, or Series
+            Index of the same type for a DatetimeIndex or TimedeltaIndex, or a Series with the same index for a Series.
+
+        Raises
+        ------
+        ValueError if the freq cannot be converted.
+
+        Notes
+        -----
+        If the timestamps have a timezone, rounding will take place relative to the local (“wall”) time and re-localized to the same timezone. When rounding near daylight savings time, use nonexistent and ambiguous to control the re-localization behavior.
+
+        Examples
+        ----------
+        DatetimeIndex
+
+        >>> rng = pd.date_range('1/1/2018 11:59:00', periods=3, freq='min')
+        >>> rng  # doctest: +SKIP
```

Comment:
@sfc-gh-nkumar in this case when calling `date_range`, the returned `DatetimeIndex` had `freq=None` rather than the expected `freq='min'`. So I'm skipping for this reason.

(I'm intentionally tagging @sfc-gh-nkumar and not @sfc-gh-nkrishna here. It's not a typo :))

@sfc-gh-nkrishna the other skips are because `round` has not been implemented yet for `DatetimeIndex`.